frame/support/procedural/src/benchmark.rs

```diff
@@ -228,17 +228,8 @@ fn parse_params(item_fn: &ItemFn) -> Result<Vec<ParamDef>> {
 		let Some(segment) = tpath.path.segments.last() else { return invalid_param(typ.span()) };
 		let args = segment.arguments.to_token_stream().into();
 		let Ok(args) = syn::parse::<RangeArgs>(args) else { return invalid_param(typ.span()) };
-		let Ok(start) = args.start.base10_parse::<u32>() else { return invalid_param(args.start.span()) };
-		let Ok(end) = args.end.base10_parse::<u32>() else { return invalid_param(args.end.span()) };
 
-		if end < start {
-			return Err(Error::new(
-				args.start.span(),
-				"The start of a `ParamRange` must be less than or equal to the end",
-			))
-		}
```

Comment:
we can probably replace this with an assertion at runtime

Reply:
Added a check.